bin/vaultclient

```diff
@@ -128,10 +128,12 @@ program
     .option('--name <NAME>')
     .option('--accesskey <ExternalAccessKey>')
     .option('--secretkey <ExternalSecretKey>')
+    .option('--expirationTime <ExpirationTime>')
```

Comment:
Is `expirationTime` the target time when the keys are meant to expire or is it the duration in number of seconds?

Reply:
```suggestion
    .option('--expirationTime <ExpirationTime>', '<<some explanation of what type of input is expected by vaultclient>>', parseInt)
```

Reply:
It's the duration in number of seconds, just added the explanation.